plugins/extractors/bigquery/bigquery.go

```diff
@@ -284,7 +311,8 @@ func (e *Extractor) buildAsset(ctx context.Context, t *bigquery.Table, md *bigqu
 			"dataset":             t.DatasetID,
 			"project":             t.ProjectID,
 			"type":                string(md.Type),
-			"partition_field":     partitionField,
```

Comment:
Is there any impact to breaking backward compatibility here? Is there a downside to maintain backward compatibility by populating it as `md.TimePartitioning.Field` when present?

Reply:
No downside. I will add changes to amke it backward compatible.

Reply:
Done